src/components/MapsSection/MapsSection.styles.tsx

```diff
@@ -77,8 +77,6 @@ export const TagButton = styled.span<{ active?: string }>`
   box-sizing: border-box;
   background-color: #ffffff95;
 
```

Comment:
Instead of removing this entirely we can only remove the not-allowed because we want to give a feedback to the user of when they can click on something, and with the cursor pointer we do that.

So a thing I would do is:

```suggestion
    cursor: ${({ active }) => (active !== "true" && "pointer")};
```